Example/WatchDemo/WatchDemoWatchKitExtension/WatchDemoApp.swift

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import FirebaseMessaging
+import FirebaseCore
+// import FirebaseRemoteConfig
+
+@main
+struct WatchDemoApp: App {
+  @WKExtensionDelegateAdaptor var extensionDelegate: MyExtensionDelegate
+
+  @SceneBuilder var body: some Scene {
+    WindowGroup {
+      NavigationView {
+        ContentView()
+      }
+    }
+
+    WKNotificationScene(controller: NotificationController.self, category: "myCategory")
+  }
+}
+
+class MyExtensionDelegate: NSObject, WKExtensionDelegate, MessagingDelegate {
+  func applicationDidFinishLaunching() {
+    FirebaseApp.configure()
```

Comment:
Can we keep this in the `init` above for consistency with other SwiftUI lifecycles, or does it need to be here?

Reply:
For this one, since we are taking advantage of the Delegate anyway, I thought it would be useful to show another way of init firebase?

I can always use init to keep the consistency. @peterfriese wdyt?

Reply:
Merge for now, but can always kick off another PR for change.